### PR TITLE
add more images of smashmouth and shrek

### LINKED ISSUE #3: Add more replacement images
There isn't a large pool of images to currently use for replacement. It'd be nice to grow this list so that there is less redundancy in the image swaps.

### PR DESCRIPTION
**Pull request by GeneralMeow**
fixes #3

### CODE REVIEW

File: smash.js [hunk 1]
```diff
@@ -16,8 +16,56 @@ function replaceAllImages(){
...
+    'https://timedotcom.files.wordpress.com/2016/07/smash-mouth-group-pic.jpg?quality=85',
+    'http://static.tvtropes.org/pmwiki/pub/images/smashmouth_8671.jpg',
+    'https://metrouk2.files.wordpress.com/2016/06/111154933.jpg?w=748&h=494&crop=1',
+    'http://www.altpress.com/images/uploads/news/underoath_SM.jpg',
+    'http://diffuser.fm/files/2016/06/Smash-Mouth.jpg?w=630&h=420&zc=1&s=0&a=t&q=89',
+    'http://cdn.boulevards.com/imgs/event_imgs/20150609184150-SmashMouth2EFM.jpg',
+    'http://www.aceshowbiz.com/images/photo/smash_mouth.jpg',
+    'http://i.dailymail.co.uk/i/pix/2016/08/29/22/37B1119900000578-3764207-image-a-56_1472507455538.jpg', 'http://www.inforum.com/sites/default/files/styles/16x9_620/public/0B5WzNj8WWJ6_djhvdlJKY1AyYkk.jpg?itok=W3oRWYnu',
+    'https://thumbs.mic.com/MzgwODkwZTY0NSMvX0FJaU1WQ2J0NEM0Zkp1clJZZUg5Y2xxRXBzPS8weDE2NToyOTE2eDE1NzcvMTYwMHg5MDAvZmlsdGVyczpmb3JtYXQoanBlZyk6cXVhbGl0eSg4MCkvaHR0cHM6Ly9zMy5hbWF6b25hd3MuY29tL3BvbGljeW1pYy1pbWFnZXMvdWFrZXBmZ2lldWNlZG1tczJucXd3ZmkzZzhteWEzYmdocXR2eW9kdmRhY3p3Z3dzNnVocGJveGQ2ZDdoNXBteS5qcGc.jpg',
+    'http://www.pngmart.com/image/25428',
```
Comment (Flamacue) [inline smash.js]: The url http://www.pngmart.com/image/25428 doesn't point to the image file. Maybe it should be http://www.pngmart.com/files/3/Shrek-PNG-Photos.png ?

File: smash.js [hunk 2]
```diff
@@ -16,8 +16,56 @@ function replaceAllImages(){
...
+    'https://cbs931jackfm.files.wordpress.com/2014/04/smash-mouth-brenda-chase-getty-images.jpg?w=640&h=360&crop=1',
+    'https://s-media-cache-ak0.pinimg.com/originals/e0/0f/de/e00fde21dee2a075d2c4456b97a8af4a.jpg',
+    'https://timedotcom.files.wordpress.com/2016/07/smash-mouth-group-pic.jpg?quality=85',
+    'http://static.tvtropes.org/pmwiki/pub/images/smashmouth_8671.jpg',
+    'https://metrouk2.files.wordpress.com/2016/06/111154933.jpg?w=748&h=494&crop=1',
+    'http://www.altpress.com/images/uploads/news/underoath_SM.jpg',
+    'http://diffuser.fm/files/2016/06/Smash-Mouth.jpg?w=630&h=420&zc=1&s=0&a=t&q=89',
+    'http://cdn.boulevards.com/imgs/event_imgs/20150609184150-SmashMouth2EFM.jpg',
+    'http://www.aceshowbiz.com/images/photo/smash_mouth.jpg',
+    'http://i.dailymail.co.uk/i/pix/2016/08/29/22/37B1119900000578-3764207-image-a-56_1472507455538.jpg', 'http://www.inforum.com/sites/default/files/styles/16x9_620/public/0B5WzNj8WWJ6_djhvdlJKY1AyYkk.jpg?itok=W3oRWYnu',
```
Comment (Flamacue) [inline smash.js]: Can you please add a newline between these two urls on line 48 for consistency? :)

**Review by Flamacue — CHANGES_REQUESTED** [review]
I've left a few comments for some minor changes. Once they're fixed, I'm more than happy to merge it, thanks! :)

File: smash.js (hunk 1 above)
Comment (GeneralMeow) [reply smash.js]: done and done

**Review by Flamacue — APPROVED** [review]

**Pull request merged**